core/src/main/resources/hudson/model/Computer/_api.jelly

```diff
@@ -43,5 +43,14 @@ THE SOFTWARE.
       Or you can POST to <a href="../changeOfflineCause?offlineMessage=some%20text">this URL</a>
       to mark this node temporary offline, or adjust the offline cause message if it is already offline.
   </p>
+    <h2>Retrieving Agent Secret</h2>
+    <p>
+        To retrieve the agent secret (JNLP MAC) via the API, you can access <a href="../agent-secret/">this URL</a>.
```

Comment:
```suggestion
        To retrieve the agent secret via the API, you can access <a href="../agent-secret/">this URL</a>.
```

User-facing docs calls it "secret" or "secret key". I don't think the audience of this documentation is helped by calling it "JNLP MAC".